File: yoroflow-ui/src/app/mytasks-module/mytasks/pagination-vo.ts
export class PaginationVO {
    index: number;
    size: number;
    direction: string;
    columnName: string;
    id: number;
    taskStatus: string;
    dueDateBoolean: boolean;
    filterValue: FilterValuesVO[] = [];
    forFilterList: boolean;
    wildSearch: string;
    taskType: string;
    taskName: string;
    filterIndex: number;
    taskNameList: any;
    filterIndexList: number[];
    taskboardId: string;
    filterColumnName: string;
    workspaceId: string;
    workspaceIdList: string[] = [];
    taskboardIdList: string[] = [];
    widgetName: string;
    startDate: any;
    endDate: any;
    filterType: string;
    allWorkspace:boolean;
}

export class FilterValuesVO {
    filterIdColumn: string;
    operators: string;
    filterDataType: string;
    filterIdColumnValue: string;
}
export class FieldValueVO {
    fieldId: string;
    fieldName: string;
    datatype: string;
    taskType: string;
}
export class ReassignTaskVO {
    instanceId: any;
    instanceTaskId: any;
    assignedToUser: any;
    assignedToGroup: any;
    status: any;
}

File: yoroflow-service/src/main/java/com/yorosis/yoroflow/models/PaginationVO.java
package com.yorosis.yoroflow.models;

import java.time.LocalDateTime;
import java.util.List;
import java.util.UUID;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PaginationVO {
	private int index;
	private int size;
	private String direction;
	private String columnName;
	private String taskStatus;
	private boolean dueDateBoolean;
	private FilterValueVO[] filterValue;
	private UUID processInstanceId;
	private String wildSearch;
	private String dateSearch;
	private LocalDateTime startDate;
	private LocalDateTime endDate;
	private String taskType;
	private String taskName;
	private int filterIndex;
	private List<Integer> filterIndexList;
	private boolean isFromTaskboard;
	private UUID taskboardId;
	private String filterColumnName;
	private UUID workspaceId;
	private List<UUID> taskboardIdList;
	private String filterType;
	private Boolean allWorkspace;
}
